src/box/vy_stmt.c

```diff
@@ -261,6 +250,14 @@ vy_stmt_dup_lsregion(struct tuple *stmt, struct lsregion *lsregion,
 	 */
 	mem_stmt->refs = 0;
 	return mem_stmt;
+
+	/*
+	 * Since n_upserts is only used in statements allocated by lsregion,
+	 * we should to initialize it here. But since n_upserts is zero
```

Comment:
should not

Reply:
I meant 'we should initialize", I'll fix it, will it be OK?